src/messages/javascript.js

```diff
@@ -141,6 +141,40 @@ export const ONLY_PREFS_IN_DEFAULTS = {
   ],
 };
 
+export const NO_DOCUMENT_WRITE = {
+  code: 'NO_DOCUMENT_WRITE',
+  message: _('Use of document.write strongly discouraged.'),
```

Comment:
I'm leaving a comment here because separate to this PR I'd like to propose some copy-edits here, but it can be a separate PR that doesn't need to block this. IGNORE ME. ^_^